snapcraft.yaml

```diff
@@ -190,7 +224,15 @@ parts:
         # (locally-built versions are more recent than the ones in the gnome platform snap)
         export LD_LIBRARY_PATH="$CRAFT_PART_BUILD/obj-$TARGET_TRIPLET/instrumented/dist/bin${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}"
       fi
-      export PATH=$PATH:/usr/lib/llvm-15/bin/
+      BUILD_DBGSYMS=false
+      if [ $CRAFT_ARCH_BUILD_ON = "amd64" ]; then
```

Comment:
For Firefox we build debug symbols also on arm64, should we do the same in Thunderbird?